crates/js-component-bindgen/src/transpile_bindgen.rs

```diff
@@ -331,6 +342,45 @@ impl Instantiator<'_, '_> {
         self.exports(&self.component.exports)
     }
 
+    fn trampoline(&mut self, i: TrampolineIndex, trampoline: &'a Trampoline) {
+        match trampoline {
+            // This is only used for a "degenerate component" which internally
+            // has a function that always traps. While this should be trivial to
+            // implement (generate a JS function that always throws) there's no
+            // way to test this at this time so leave this unimplemented.
+            Trampoline::AlwaysTrap => unimplemented!(),
+
+            // This is required when strings pass between components within a
+            // component and may change encodings. This is left unimplemented
+            // for now since it can't be tested and additionally JS doesn't
+            // support multi-memory which transcoders rely on anyway.
+            Trampoline::Transcoder {
+                op: _,
+                from: _,
+                from64: _,
+                to: _,
+                to64: _,
+            } => unimplemented!(),
```

Comment:
Is this path then effectively skipped in the polyfill case?

Reply:
Nah this'll still needs to get filled out, but I wanted to tackle this PR first as a first step before tackling strings